terraform/software/README.md

```diff
@@ -40,6 +42,7 @@ No resources.
 | <a name="input_enable_cert_manager"></a> [enable\_cert\_manager](#input\_enable\_cert\_manager) | Enables the use of cert manager. | `bool` | `false` | no |
 | <a name="input_enable_http_adapter"></a> [enable\_http\_adapter](#input\_enable\_http\_adapter) | Used to enable the http adapter | `bool` | `false` | no |
 | <a name="input_enable_mqtt_adapter"></a> [enable\_mqtt\_adapter](#input\_enable\_mqtt\_adapter) | Used to enable the mqtt adapter | `bool` | `true` | no |
+| <a name="input_enhanced_mqtt_load_balancer"></a> [enhanced\_mqtt\_load\_balancer](#input\_enhanced\_mqtt\_load\_balancer) | Configuration options for the enhanced MQTT load balancer.<br>  enabled: Enables the use of the enhanced MQTT load balancer.<br>  chart\_version: Version of the chart to deploy.<br>  algorithm: Load balancing algorithm used by the enhanced MQTT load balancer. For a list of possible options see https://www.haproxy.com/documentation/kubernetes-ingress/community/configuration-reference/ingress/#load-balance .<br>  replicaCount: Number of replicas to deploy.<br>  port\_configs: List of MQTT port config objects for the enhanced MQTT load balancer service.<br>  tcp\_configmap\_data: Data of the TCP configMap for the enhanced MQTT load balancer. | <pre>object({<br>    enabled = optional(bool, true),<br>    chart_version = optional(string, "1.34.1"),<br>    algorithm = optional(string, "leastconn"),<br>    replicaCount = optional(number, 1),<br>    port_configs = optional(list(object({<br>      name       = string,<br>      port       = number,<br>      targetPort = optional(number, 8883)<br>    })), [<br>            {<br>              name: "mqtt"<br>              port: 8883<br>              targetPort: 8883<br>            }<br>          ]),<br>    tcp_configmap_data = optional(map(string), {<br>            8883 = "hono/eclipse-hono-adapter-mqtt:8883"<br>          })<br>  })</pre> | `{}` | no |
```

Comment:
The description including the link (explaination) should be moved out of the table if possible. Can we maybe create a new section with a table where we describe it? link should be in a description outside the table if possible

Reply:
I would prefer not to change anything here, as this readme is auto generated